addons/service/multimedia/streamdisc-server/source/bin/streamdisc-server.start

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+. /etc/profile
+
+oe_setup_addon service.multimedia.streamdisc-server
+
+ADDON_SETTINGS="$ADDON_HOME/settings.xml"
+
+chmod a+x $ADDON_DIR/bin/*
+
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$ADDON_DIR/lib"
+
+exec $ADDON_DIR/bin/streamdisc_server $STREAMDISC_PORT $STREAMDISC_DEVICE
```

Comment:
STREAMDISC_PORT and STREAMDISC_DEVICE are supposed to come from the 
addon settings (source/settings_default.xml, 
source/resources/settings.xml) and are set by

oe_setup_addon service.multimedia.streamdisc-server. Works on my system...

On 11/27/15 21:13, Lukas Rusak wrote:
>
> In 
> addons/service/multimedia/streamdisc-server/source/bin/streamdisc-server.start 
> <https://github.com/OpenELEC/unofficial-addons/pull/88#discussion_r46068766>:
>
> > +#
> > +#  You should have received a copy of the GNU General Public License
> > +#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
> > +################################################################################
> > +
> > +. /etc/profile
> > +
> > +oe_setup_addon service.multimedia.streamdisc-server
> > +
> > +ADDON_SETTINGS="$ADDON_HOME/settings.xml"
> > +
> > +chmod a+x $ADDON_DIR/bin/*
> > +
> > +export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$ADDON_DIR/lib"
> > +
> > +exec $ADDON_DIR/bin/streamdisc_server $STREAMDISC_PORT $STREAMDISC_DEVICE
>
> Where does STREAMDISC_PORT and STREAMDISC_DEVICE come from?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OpenELEC/unofficial-addons/pull/88/files#r46068766>.
>

-- 
---------------------------
tilman wolff-siemssen
kopischstr. 3
10965 berlin

fon +49 179 900 1074